Assets/MixedRealityToolkit/_Core/Definitions/Devices/TrackingState.cs

```diff
@@ -4,34 +4,26 @@
 namespace Microsoft.MixedReality.Toolkit.Internal.Definitions.Devices
 {
     /// <summary>
-    /// The Controller State defines how a controller or headset is currently being tracked.
+    /// The Tracking State defines how a controller or headset is currently being tracked.
     /// This enables developers to be able to handle non-tracked situations and react accordingly
     /// </summary>
-    public enum ControllerState
+    public enum TrackingState
     {
         /// <summary>
-        /// No  controller state provided by the SDK.
+        /// The controller is currently not tracked.
         /// </summary>
-        None = 0,
+        NotTracked = 0,
         /// <summary>
-        /// Reserved, for systems that provide alternate tracking.
+        /// The controller is tracked, but has approximate positioning.
         /// </summary>
-        Other,
+        Approximate,
```

Comment:
Not needed with #2347 (we may need to support approximate orientation on some platform)

Reply:
I'd still like to keep it in case we will need it on other platforms.  I'm fine with removing position and rotation only, but this will stay.